braindecode/samplers/base.py

```diff
@@ -115,7 +119,11 @@ class SequenceSampler(RecordingSampler):
     n_windows : int
         Number of consecutive windows in a sequence.
     n_windows_stride : int
-        Number of windows between two consecutive sequences.
+        Number of windows between two consecutive starts of sequences.
+        n_windows_stride=1 is maximal overlap.
+        n_windows_stride=n_windows is minimal overlap.
```

Comment:
minimal overlap meaning no overlap or a single overlapping window? assume it is no overlap right? then maybe better to write like that?